Makefile

```diff
@@ -82,7 +82,19 @@ vagrant-deploy: vagrant-sync-config vagrant-sync-build
 .release-functest:
 	make functest > .release-functest 2>&1
 
+kubevirt.yaml: TAG=$(shell git describe) PREFIX=kubevirt
+kubevirt.yaml:
+	echo "docker_prefix=$(PREFIX)" > hack/config-local.sh
+	echo "docker_tag=$(TAG)" >> hack/config-local.sh
+	./hack/build-manifests.sh
+	echo "# Created on $(shell date) at $(shell git describe --always)" > $@
+	echo "# For tag '$(TAG)'" >> $@
+	for M in manifests/*.yaml; do cat $$M ; echo "---" ; done >> $@
```

Comment:
I think we should consider building two manifests.

Devel: would include things like the storage demo services we have.
Production: stripped down to only kubevirt runtime services/components.

Reply:
Good point.

Reply:
To be honest I would propose to go for three since test environment usually differs from the dev and from the production.